src/daily_output.ts

```diff
@@ -0,0 +1,474 @@
+import { readFileSync, writeFileSync } from 'fs';
+import { GROUP_EFFORT_REWARDS, ONE_THOUSAND_MB, OUTPUT_NAME, OUTPUT_TEMPLATE_NAME } from './constants';
+import { GQLEdgeInterface } from './gql_types';
+import { OutputData, StakedPSTHolders, WalletsStats, WalletStatEntry } from './inferno_types';
+
+/**
+ * A class responsible of parsing the GQL and CommunityOracle data into the OutputData file
+ */
+export class DailyOutput {
```

Comment:
Right in the DocBlock I can tell this class has too many responsibilities. As far as this application is concerned, this is the god class (:

We should consider splitting this up if we expand the program, I think we're okay for now.